### PR TITLE
Issue 1477: Updated main readme, doc structure, and secret key md

### PR DESCRIPTION
**Pull request by ADPennington**
### Summary of Changes

_Pull request addresses major changes to repo doc organization, as noted in #1477_:

- [Main readme](https://github.com/raft-tech/TANF-app/blob/3556117d30db4d26c087787ee6019122adc8bc7a/README.md) updated per [hackmd](https://hackmd.io/gqdLDoTXQvysIO4HMFfQNQ)
- `TANF-app/design-assets/` moved to `TANF-app/docs/User-Experience/Design/design-assets/` 
- Updated `TANF-app/docs/` as follows:
> TANF-app/docs/
> > > Background/
> > > Product-Strategy/
> > > How-We-Work/
> > > Security-Compliance/
> > > Sprint-Review/
> > > Technical-Documentation/
> > > User-Experience/

- `TANF-app/docs/Acquisition/` moved into `TANF-app/docs/Background/` 
- `TANF-app/docs/Prototype/` moved into `TANF-app/docs/Background/` 
- `TANF-app/docs/Design/` moved into `TANF-app/docs/User-Experience/` 
- ADR subdir moved into `./docs/Technical-Documentation/` (and is consistent with file changes in #1488  though subdir and file names have been changed)
- `Secrey-Key-Mgmt.md` renamed and moved into `TANF-app/docs/Security-Compliance/` for now. 

### Important note(s):
- no other files within the repo have moved, but internal reference links in markdowns are likely to be broken upon merge and will need to be updated in a follow-on PR. 
- #1477 stub needs to be updated to reflect where files should be moved per comments therein.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,148 +1,70 @@
...
 
 ## Infrastructure
 
 TDP Uses Infrastructure as Code (IaC) and DevSecOps automation
 
 ### Authentication
 
-[Login.gov](https://login.gov/) TDP requires strong multi-factor authentication for the states, tribes, and territories and Personal Identity Verification (PIV) authentication for OFA staff. Login.gov is being used to meet both of these requirements. 
+TDP application requires strong multi-factor authentication (MFA) for all users, and Personal Identity Verification (PIV) authentication must be used as the 2nd factor for all internal ACF staff. 
+[ACF AMS]() authentication service is being used for ACF users, and [NextGen XMS]()authentication service is being used for external users.
```
Comment (ADPennington) [inline README.md:42]:
```suggestion
[ACF AMS]() authentication service is being used for ACF users, and [NextGen XMS]() authentication service is being used for external users. 
```

**Comment by ADPennington** [comment]
ready for review: @lfrohlich @valcollignon @abottoms-coder. if these changes suffice, 1505 can be closed in draft.

**Review by lfrohlich — APPROVED** [review]
LGTM 👍

**Review by valcollignon — APPROVED** [review]
LGTM!

**Review by sreedevip — APPROVED** [review]
Looks great

File: docs/Technical-Documentation/Architecture-Decision-Record/009-git-workflow.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+Date: 2021-02-23 (_updated 2022-01-06_)
+
+## Status
+
+Accepted
+
+## Context
+
+In order to maintain the principal of Least Privilege, it was decided at the onset of this project that the vendor (Raft) would work from a fork of the government repo and issue pull requests to the government repo from the fork. The vendor would not have write access to the government repository or the government's CircleCI account.
```
Comment (ADPennington) [inline docs/Technical-Documentation/Architecture-Decision-Record/009-git-workflow.md:11]:
```suggestion
In order to maintain the principle of Least Privilege, it was decided at the onset of this project that the vendor (Raft) would work from a fork of the government repo and issue pull requests to the government repo from the fork. The vendor would not have write access to the government repository or the government's CircleCI account.
```

File: docs/Technical-Documentation/Architecture-Decision-Record/009-git-workflow.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+- Less complexity for contributers and reviewers
+- No need to rebase from `HHS:main` back to `raft-tdp-main`
+- Much smaller chance of needing to revert commits to `HHS:main`
+- Git history will be much cleaner
+- Much less time managing git history
+- All reviews will be in one place, so the entire history of a branch/PR will be viewable in one place
+- Will allow us to keep the current CircleCi setup
+  - Once staging is in place, we can deploy to the development server when code is merged in to `raft-tdp-main` and test deployment ahead of opening a PR to `HHS:main`
+- We can implement GitHub Hooks to automatically issue PRs to `HHS:main` when one is merged to `raft-tdp-main`
+- Maintain "Least Privelege" by restricting vendor from having write access to Gov repo and CircleCI
```
Comment (ADPennington) [inline docs/Technical-Documentation/Architecture-Decision-Record/009-git-workflow.md:92]:
```suggestion
- Maintain "Least Privilege" by restricting vendor from having write access to Gov repo and CircleCI
```

**Comment by abottoms-coder** [comment]
@ADPennington Is this a work in progress pull request? I see that there are new commits and wasn't sure if those were a result of changes requested offline.

**Comment by ADPennington** [comment]
> @ADPennington Is this a work in progress pull request? I see that there are new commits and wasn't sure if those were a result of changes requested offline.

no its done. had to make a spelling fix per @sreedevip. its ready. also this gitflow ADR is more up to date than in 1505. @abottoms-coder

**Comment by abottoms-coder** [comment]
> > @ADPennington Is this a work in progress pull request? I see that there are new commits and wasn't sure if those were a result of changes requested offline.
> 
> no its done. had to make a spelling fix per @sreedevip. its ready. also this gitflow ADR is more up to date than in 1505. @abottoms-coder

Once 1082 PR is merged, would like to see this PR status with new changes from `raft-tdp-main`. Otherwise ready to approve and move forward.

**Review by abottoms-coder — APPROVED** [review]

**Pull request merged**